projects/ion/src/lib/alert/alert.component.scss

```diff
@@ -47,11 +58,33 @@
   @include alert-style($negative-1, $negative-6);
 }
 
+.with-description {
+  border: none !important;
+  background-color: $neutral-2;
+  padding: 16px 24px !important;
+  color: $neutral-7;
+  gap: 4px;
+
+  .message-text {
+    font-weight: 600;
+  }
+
+  .description {
+    word-break: break-all;
+    hyphens: auto;
```

Comment:
I think I removed these properties when submited the review, you can see on the other comment.